src/modals/confirm-sign-modal/use-sign-message.hook.ts

```diff
@@ -72,8 +72,8 @@ export const useSignMessage = (formValues: CreateNftFormValues) => {
               })
             )
           ),
-          catchError(() => {
-            showWarningToast({ description: 'Ooops, something went wrong.\nPlease, try again later.' });
+          catchError(e => {
+            showErrorToast({ description: getAxiosQueryErrorMessage(e) });
```

Comment:
I would rather show an error message from backend if it returns 400-499 status. The message is available under `message` or `error` key (see https://stable-diffusion-nft-backend.stage.madfish.xyz/docs/#/default/post_api_v1_orders for example). Unfortunately, `getAxiosQueryErrorMessage` doesn't get it but returns something like `Request to <URL> failed with status <status>`. It also would be nice to make the message from backend copyable in all cases.